tket/README.md

```diff
@@ -60,88 +40,65 @@ cd ${PKGPATH}/bin
 cd -
 ```
 
-To build the property tests, writing package information to
-`proptest-tket.json`:
+The tests with a running time >=1 second (on a regular modern laptop) are marked
+as hidden, tagged with `"[long]"`, and are not run by default. To run the long
+tests use the `"[long]"` tag as an argument:
 
 ```shell
-conan create --profile=tket tket/proptest --build=missing --json proptest-tket.json
+./test-tket "[long]"
 ```
 
-To extract the root package path to environment variable:
-
-```shell
-PKGPATH=`./rootpath proptest-tket.json proptest-tket`
-```
-
-To run the property tests:
+To run the full suite manually you need to include also the short tests, like:
 
 ```shell
-cd ${PKGPATH}/bin
-./proptest-tket
-cd -
-```
-
-## conan 2
-
-```shell
-pip install conan~=2.0
-```
-
-Use the default profile:
-
-```
-conan profile detect
+./test-tket "[long],~[long]"
 ```
 
-Add the remote:
+A smaller selection of the compiled tests can also be run by passing a filter of
+the test file name:
 
 ```shell
-conan remote add tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs
+./test-tket -# "[#test_name]"
 ```
 
-To build `tket`:
-
-```shell
-conan create tket --user=tket --channel=stable --build=missing -o boost/*:header_only=True
-```
-(When using `zsh` shell, consider adding quotes around `boost/*`.)
 
-To build the unit tests:
+To build the property tests:
 
 ```shell
-conan create tket/test --build=missing  -o boost/*:header_only=True --format json > test-tket.json
+conan create tket/proptest --build=missing  -o boost/*:header_only=True --format json > proptest-tket.json
 ```
 
 To extract the root package path to environment variable:
 
 ```shell
-PKGPATH=`./rootpath test-tket.json test-tket`
+PKGPATH=`./rootpath proptest-tket.json proptest-tket`
 ```
 
-To run the unit tests:
+To run the property tests:
 
 ```shell
 cd ${PKGPATH}/bin
-./test-tket
+./proptest-tket
 cd -
 ```
 
-To build the property tests:
-
-```shell
-conan create tket/proptest --build=missing  -o boost/*:header_only=True --format json > proptest-tket.json
-```
+## Generating a test coverage report
 
-To extract the root package path to environment variable:
+If using gcc and with gcovr installed (`pip install gcovr`), a test coverage
```

Comment:
I am only able to get this to work without gcovr from python and with the global installation. Should we change this here? If this is working for you I am happy to keep it as it is and just will keep this in mind

Reply:
Hmm, works for me with `gcovr` installed using `pip`.

Reply:
(Works on the CI too...)

Reply:
Do you have a global installation with the same version?

Reply:
If that is the way it works on the CI lets keep it as it is :)

Reply:
> Do you have a global installation with the same version?

No I don't have a version installed outside of my python environment.